enclave/src/openfinex/openfinex_client.rs

```diff
@@ -159,9 +165,10 @@ impl TcpClient {
             debug!(
                 "Received Http response: {}",
                 String::from_utf8_lossy(&buffer)
-            ); //FIXME: replace with debug
-               // handshake successful, send a subscription:
+            );
+            // handshake successful, send a subscription:
             self.subscribe_matches();
+            self.send_markets_request();
```

Comment:
After we get a successful handshake, we send the request for the market information